### PR TITLE
using maven bom

### PR DESCRIPTION
**Pull request by paulvi**
started in #49

### CODE REVIEW

File: source/getting-started/how-to-create-a-struts2-web-application.md [hunk 1]
```diff
@@ -176,6 +176,52 @@ Using both `log4j-core` and `log4j-api` allows to use [the latest version of Log
...
+<dependencies>
+	<!-- Struts 2 -->
+	<dependency>
+		<groupId>org.apache.struts</groupId>
+		<artifactId>struts2-core</artifactId>
+	</dependency>
+
+	<dependency>
+		<groupId>org.apache.logging.log4j</groupId>
+		<artifactId>log4j-core</artifactId>
```
Comment (lukaszlenart) [inline source/getting-started/how-to-create-a-struts2-web-application.md]: I would add `log4j-api` as well

Comment (paulvi) [reply source/getting-started/how-to-create-a-struts2-web-application.md]: log4j-api is already managed in second bom import.  
So no need to specify it just to put correct version.

I just tried it today, no need.

Comment (paulvi) [reply source/getting-started/how-to-create-a-struts2-web-application.md]: There is other problem as the example will have red problem in Eclipse IDE for `index.jsp`,  until servlet dependency is added
```xml
		<dependency>
			<groupId>javax.servlet</groupId>
			<artifactId>servlet-api</artifactId>
			<version>2.4</version>
			<scope>provided</scope>
		</dependency>
```

Comment (paulvi) [reply source/getting-started/how-to-create-a-struts2-web-application.md]: image

Comment (lukaszlenart) [reply source/getting-started/how-to-create-a-struts2-web-application.md]: Hm... up to you if the `servlet-api` should be defined in the `pom.xml`

Comment (paulvi) [reply source/getting-started/how-to-create-a-struts2-web-application.md]: servlet-api would be out of this PR

**Comment by lukaszlenart** [comment]
👍

**Pull request merged**